mula/tests/integration/test_api.py

```diff
@@ -925,3 +925,11 @@ def test_patch_schedule_validate_malformed_schedule(self):
         response = self.client.patch(f"/schedules/{str(self.first_schedule.id)}", json={"schedule": "malformed"})
         self.assertEqual(400, response.status_code)
         self.assertIn("validation error", response.json().get("detail"))
+
+    def test_delete_schedule(self):
+        response = self.client.delete(f"/schedules/{str(self.first_schedule.id)}")
```

Comment:
Not a big deal but: the `str` casts in the f-string is not needed, every argument will be implicitly stringified